pkgm.ts

```diff
@@ -1,21 +1,31 @@
 #!/usr/bin/env -S pkgx --quiet deno^2.1 run --ext=ts --allow-sys=uid --allow-run --allow-env=PKGX_DIR,HOMEBREW_PREFIX,HOME --allow-read=/usr/local/pkgs
 import { dirname, fromFileUrl, join } from "jsr:@std/path@^1";
 import { ensureDir, existsSync } from "jsr:@std/fs@^1";
-import { parse as parse_args } from "jsr:@std/flags@0.224.0";
+import { parseArgs } from "jsr:@std/cli@^1";
 import * as semver from "jsr:@std/semver@^1";
 
 function standardPath() {
-  const basePath = "/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin";
-  // for pkgm installed via homebrew
-  const homebrew = `${Deno.env.get("HOMEBREW_PREFIX") || "/opt/homebrew"}/bin`;
-  if (Deno.build.os === "darwin") {
-    return `${homebrew}:${basePath}`;
-  } else {
-    return basePath;
+  let path = "/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin";
+
+  // for pkgx installed via homebrew
+  let homebrewPrefix = "";
+  switch (Deno.build.os) {
+    case "darwin":
+      homebrewPrefix = "/opt/homebrew"; // /usr/local is already in the path
+      break;
+    case "linux":
+      homebrewPrefix = `/home/linuxbrew/.linuxbrew:${Deno.env.get("HOME")}/.linuxbrew`;
+      break;
   }
+  if (homebrewPrefix) {
+    homebrewPrefix = Deno.env.get("HOMEBREW_PREFIX") ?? homebrewPrefix;
+    path = `${homebrewPrefix}/bin:${path}`;
+  }
+
+  return path;
```

Comment:
it would be slower to use `fs.exists()`, but we could just loop an array of paths and add them if they're there.

Reply:
Yeah, but I think it still makes sense to only search on some directories conditionally. For example, it doesn't make sense to search for `/home/linuxbrew/.linuxbrew` on a macOS.

Reply:
that's reasonably correct. if it's present, who knows what it's doing?